updater/lib/dependabot/updater/group_update_creation.rb

```diff
@@ -93,9 +113,9 @@ def compile_all_dependency_changes_for(group)
       ensure
         cleanup_workspace
```

Comment:
I think I might be hitting a bug with Sorbet, where this `ensure` block is returning `T::Private::Types::Void::VOID` instead of `dependency_change`